common/modifiers/hmo_hardcoded.txt

```diff
@@ -200,23 +200,24 @@ nation_rank_5 = {
 	monthly_wage_modifier = 0.10
 	max_mercenary_stacks = 4
 
-	agressive_expansion_impact = 0.2
-	agressive_expansion_monthly_change = 0.05
+	agressive_expansion_impact = 0.3
+	agressive_expansion_monthly_change = 0.075
 }
 
 is_tribe = {
 	max_loyalty = 100
 	enslavement_efficiency = 0.025
 	change_governor_policy_cost_modifier = 0.1
-	army_maintenance_cost = 0.25
+	army_maintenance_cost = 0.3
 	global_tribesmen_desired_pop_ratio = 0.1
+	hill_fort_cost = -0.25
 	price_found_city_cost_modifier = 0.25
 	price_revoke_city_status_cost_modifier = -0.2
     price_revoke_metropolis_status_cost_modifier = -0.2
-	monthly_military_experience = 0.30
+	monthly_military_experience = 0.2
 	global_defensive = -0.25
-	#buy_military_tradition_cost_modifer = -0.1
-
+	land_morale = 0.375 # equal to 5 levels of military tech
+	experience_decay = -0.15 # this will get removed by tribal legion laws once enacted, so makes tribal levies better until they get legions
```

Comment:
since tribes don't get legions for a long while, make their levy xp retention better by default